meteor/client/ui/Account/NotLoggedIn/LoginPage.tsx

```diff
@@ -10,8 +10,8 @@ import { getUser, User } from '../../../../lib/collections/Users'
 import { NotLoggedInContainer } from './lib'
 import { stringifyError } from '@sofie-automation/corelib/dist/lib'
 
-interface ILoginProps extends RouteComponentProps<{ token: string }> {
-	requestedRoute: string
+interface ILoginProps extends RouteComponentProps<{ token?: string }> {
```

Comment:
I think it should technically be optional. This view behaves differently depending on if a token is provided (which means it's in a "verify-email" mode) and if it's not provided (regular login). So I think it makes sense to say it's optional, because it's a mode-switch.